api-tests/apply_route_test.go

```diff
@@ -332,6 +334,8 @@ func TestApplyPsmdbReturnedVersions(t *testing.T) {
 		{"latest", "1.7.0", nil, "4.4.3-5"},
 		{"latest", "1.6.0", nil, "4.4.2-4"},
 		{"latest", "1.5.0", nil, "4.2.8-8"},
+		{"latest", "1.14.0", &v50, "5.0.14-12"},
```

Comment:
We need to add 5.0 to 1.13 and 1.12 because we had it there. In 1.11 it was not recommended so I don't think we need to add it there.